README.md

```diff
@@ -58,20 +56,6 @@ Then [bundle](https://github.com/hyperapp/hyperapp/blob/master/docs/getting-star
 import { router, Link } from "@hyperapp/router"
 ```
 
-## Mixin
-
-Use the router as any other [mixin](https://github.com/hyperapp/hyperapp/blob/master/docs/mixins.md). Then compose your view as an array of [routes](#routes).
-
-```jsx
-app({
-  view: [
-    ["/", state => <h1>Hi.</h1>]
-    ["*", state => <h1>404</h1>],
-  ],
-  mixins: [router()]
-})
-```
-
 ### Routes
```

Comment:
@speier Maybe remove all this outdated stuff?

Reply:
👍